frontend/test/createProjectForm.test.tsx

```diff
@@ -0,0 +1,55 @@
+import "@testing-library/jest-dom";
+import { render, screen, waitFor } from "@testing-library/react";
+import mockAxios from "jest-mock-axios";
+import { AxiosResponse } from "axios";
+import apiPaths from "../src/properties/apiPaths";
+import CreateProject from "../src/pages/projects/create";
+import {
+    createProjectSubmitHandler,
+    ProjectCreationValues,
+} from "../src/handlers/createProjectSubmitHandler";
+import { getBaseOkResponse } from "./TestEntityProvider";
+
+jest.mock("next/router", () => require("next-router-mock"));
+
+afterEach(() => {
+    mockAxios.reset();
+});
+
+describe("CreateProjectForm initialization", () => {
+    it("Should call axios.get() upon rendering", () => {
+        render(<CreateProject />);
+        expect(mockAxios.get).toHaveBeenCalled();
+
+        expect(screen.getByTestId("projectname-input")).toBeInTheDocument();
+        expect(screen.getByTestId("projectinfo-input")).toBeInTheDocument();
+        expect(screen.getByTestId("versionmanagement-input")).toBeInTheDocument();
+        expect(screen.getByTestId("partnername-input")).toBeInTheDocument();
+        expect(screen.getByTestId("partnerwebsite-input")).toBeInTheDocument();
+        expect(screen.getByTestId("skillinfo-input")).toBeInTheDocument();
+        expect(screen.getByTestId("coach-input")).toBeInTheDocument();
+        expect(screen.getByTestId("skill-input")).toBeInTheDocument();
+    });
+});
+
+it("SubmitHandler for createProject sends post request", async () => {
+    const values: ProjectCreationValues = {
+        projectName: "Test project",
+        projectInfo: "This is a test project",
+        versionManagement: "https://github.com/Test",
```

Comment:
You might want to use `http://www.example.com/` since it is used for these kinds of things.

Reply:
(The url you picked takes you to a github user)